Client/Frontend/Browser/Browser.swift

```diff
@@ -170,11 +170,12 @@ class Browser: NSObject {
             }
 
             if ErrorPageHelper.isErrorPageURL(url) {
-                let decodedURL = ErrorPageHelper.decodeURL(url)
-                if !AboutUtils.isAboutURL(decodedURL) {
-                    return decodedURL
-                } else {
-                    return nil
+                if let decodedURL = ErrorPageHelper.decodeURL(url) {
```

Comment:
We shouldn't need an `if let` here. Reading your fix, it looks like `ErrorPageHelper.decodeURL(url)` is returning a bogus error URL, which then gets returned via `decodedURL`? If that's the case, we should fix this bug by fixing the broken method (`decodeURL`), not here.